test/valid_field_test.exs

```diff
@@ -23,6 +23,11 @@ defmodule ValidFieldTest do
       ValidField.with_changeset(%Model{})
       |> ValidField.assert_invalid_field(:title, ["", nil, "Something else"])
     end
+
+    assert_raise ValidField.ValidationError, "Expected the following values to be invalid for \"date_of_birth\": #Ecto.Date<2016-08-23>", fn ->
+      ValidField.with_changeset(%Model{})
```

Comment:
Even though some of the other code is written differently we are aligning with @chrismccord's recommendation on piping. We start with the inner most value and continue to pipe. This would be rewritten as:

```elixir
%Model{}
|> ValidField.with_changeset()
|> |> ValidField.assert_invalid_field(:date_of_birth, [Ecto.Date.utc()])
```

Our styleguide is evolving and we're trying to push new PRs in this direction. Thanks!